tests/php/Model/EditableFormFieldTest.php

```diff
@@ -168,6 +168,28 @@ public function testFileFieldAllowedExtensionsBlacklist()
         $this->assertNotContains('jpg', $formField->getValidator()->getAllowedExtensions());
     }
 
+    /**
+     * Test that if folder is not set or folder was removed,
+     * then getFormField() saves file in protected folder
+     */
+    public function testFileFieldSaveInFolder()
+    {
+        $fileField_1 = $this->objFromFixture(EditableFileField::class, 'file-field-without-folder');
```

Comment:
```suggestion
        $fileField1 = $this->objFromFixture(EditableFileField::class, 'file-field-without-folder');
```
Or `$fileFieldA`

Don't want underscores in var names

Also update all the other variables in this method